app_changelog.go

```diff
@@ -0,0 +1,5 @@
+package versionbundle
+
+type AppsChangelogs map[string][]Changelog
```

Comment:
And I guess this should be like:

```
type AppsChangelogs map[string]Changelog

type Changelog []ChangelogEntry
```

But if this is a log of breaking changes then ignore me.

Reply:
@kopiczko Thanks for suggestions. There's `Changelog` already in this package.  https://github.com/giantswarm/versionbundle/blob/cd94540b7d5a064d0b106387f85078c0031271d9/changelog.go#L40 I'm not aware of better naming on this variable, please let me know if you have others.

Reply:
Yeah I thought so. It's fine.